Makefile

```diff
@@ -7,9 +7,16 @@ shoots/clean:
 	rm shoots/*transferred
 	rm shoots/*slice*
 
+# Slice a given input file into manageable chunks, so that you can run them through the
+# transfer process separately without overwhelming the target system.
+# The right number for archivematica is probably about 20.
+
+%.sliced: %
+	split -l 20 $< $<.
+
 # Request the Glacier restoration of the shoots in the given file
 # The file is expected to contain one shoot identifier per line.
-# In order to run this, set your AWS profile to one with authority in the workflow account.
+# In order to run this, set your AWS profile to one with authority in the platform account.
```

Comment:
The `[wellcomecollection-editorial-photography](https://eu-west-1.console.aws.amazon.com/s3/buckets/wellcomecollection-editorial-photography?region=eu-west-1)` is in the platform account